packages/api/server/ws.mts

```diff
@@ -99,11 +107,8 @@ async function cellExec(payload: CellExecPayloadType) {
       jsExec({ session, cell, secrets });
       break;
     case 'typescript':
-      tscExec({ session, cell, secrets });
       tsxExec({ session, cell, secrets });
       break;
-    default:
```

Comment:
This didn't have a lot of downsides, and would help discover bugs later.

Reply:
I don't really care either way, but the type system will catch it. So unless we forget to change the typesystem when adding a new language (and how often will that happen?), then I'm not sure how this helps discover bugs faster.